app/src/main/res/values/strings.xml

```diff
@@ -1068,4 +1070,14 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_camera_type_dome">Dome</string>
     <string name="quest_camera_type_fixed">Fixed</string>
     <string name="quest_camera_type_panning">Panning</string>
+    <string name="no_search_results">The what now?</string>
+    <string name="quest_selection_hint_title">Too much to do?</string>
```

Comment:
> Although "Too much to do?"

That's pretty much German ("Zu viel zu tun?") literally translated into English. If it doesn't work that well in English, please post a suggestion!

_Originally posted by @westnordost in https://github.com/streetcomplete/StreetComplete/pull/3210#discussion_r695644574_

Yeah it works fine in English, I just wonder if there is a bit of a negative connotation about work or it being a job (rather than the gamification side of things). i.e. these are your tasks to complete.

This seems a little softer or just ambiguous (regarding why you're completing the quests) to me:
```suggestion
    <string name="quest_selection_hint_title">Too many quests?</string>
```

Or alternatively to put a more positive spin on it:
```suggestion
    <string name="quest_selection_hint_title">Too much to choose from?</string>
```

Reply:
Hmm well, the point is that it should not feel like work and this very dialog addresses that. No point in denying that it may feel like work if there is too much to do:

> Too much to do?
>
> Contributing should primarily be fun!
> 
> If you are overwhelmed by the number of quests, you can always fine-tune which quests are displayed to you and in what order in the settings.</string>